official-antistasi-community/A3-Antistasi
# Meer template selector

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    Moved all template selection to it's own file, taking it out of initVarServer and the templates themselves.

### Please specify which Issue this PR Resolves.
closes #1208 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [x] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
  Load each modset and map combination and check that the correct units spawn.
********************************************************
Notes:
  I tested the vanilla maps on vanilla, RHS and 3CB and all worked fine, any modded maps should work properly as should IFA if it gets reintegrated. It should fail safe if a map's worldname is wrong by defaulting to the Arid templates.
